suave/devenv/blockscout/envs/common-blockscout.env

```diff
@@ -0,0 +1,240 @@
+# INDEXER_COIN_BALANCES_BATCH_SIZE=
+# INDEXER_COIN_BALANCES_CONCURRENCY=
+# INDEXER_RECEIPTS_BATCH_SIZE=
+# INDEXER_RECEIPTS_CONCURRENCY=
+# INDEXER_TOKEN_CONCURRENCY=
+# INDEXER_TOKEN_BALANCES_BATCH_SIZE=
+# INDEXER_TOKEN_BALANCES_CONCURRENCY=
+# INDEXER_TX_ACTIONS_ENABLE=
+# INDEXER_TX_ACTIONS_MAX_TOKEN_CACHE_SIZE=
+# INDEXER_TX_ACTIONS_REINDEX_FIRST_BLOCK=
+# INDEXER_TX_ACTIONS_REINDEX_LAST_BLOCK=
+# INDEXER_TX_ACTIONS_REINDEX_PROTOCOLS=
+# INDEXER_TX_ACTIONS_AAVE_V3_POOL_CONTRACT=
+# INDEXER_POLYGON_EDGE_L1_RPC=
+# INDEXER_POLYGON_EDGE_L1_EXIT_HELPER_CONTRACT=
+# INDEXER_POLYGON_EDGE_L1_WITHDRAWALS_START_BLOCK=
+# INDEXER_POLYGON_EDGE_L1_STATE_SENDER_CONTRACT=
+# INDEXER_POLYGON_EDGE_L1_DEPOSITS_START_BLOCK=
+# INDEXER_POLYGON_EDGE_L2_STATE_SENDER_CONTRACT=
+# INDEXER_POLYGON_EDGE_L2_WITHDRAWALS_START_BLOCK=
+# INDEXER_POLYGON_EDGE_L2_STATE_RECEIVER_CONTRACT=
+# INDEXER_POLYGON_EDGE_L2_DEPOSITS_START_BLOCK=
+# INDEXER_POLYGON_EDGE_ETH_GET_LOGS_RANGE_SIZE=
+# INDEXER_REALTIME_FETCHER_MAX_GAP=
+# INDEXER_FETCHER_INIT_QUERY_LIMIT=
+# INDEXER_TOKEN_BALANCES_FETCHER_INIT_QUERY_LIMIT=
+# INDEXER_COIN_BALANCES_FETCHER_INIT_QUERY_LIMIT=
+# INDEXER_DISABLE_WITHDRAWALS_FETCHER=
+# WITHDRAWALS_FIRST_BLOCK=
+# TOKEN_ID_MIGRATION_FIRST_BLOCK=
+# TOKEN_ID_MIGRATION_CONCURRENCY=
+# TOKEN_ID_MIGRATION_BATCH_SIZE=
+# INDEXER_INTERNAL_TRANSACTIONS_TRACER_TYPE=
+# WEBAPP_URL=
+# API_URL=
+SHOW_ADDRESS_MARKETCAP_PERCENTAGE=true
+CHECKSUM_ADDRESS_HASHES=true
+CHECKSUM_FUNCTION=eth
+DISABLE_EXCHANGE_RATES=true
+TXS_STATS_ENABLED=true
+SHOW_PRICE_CHART=false
+SHOW_PRICE_CHART_LEGEND=false
+SHOW_TXS_CHART=true
+TXS_HISTORIAN_INIT_LAG=0
+TXS_STATS_DAYS_TO_COMPILE_AT_INIT=10
+COIN_BALANCE_HISTORY_DAYS=90
+APPS_MENU=true
+EXTERNAL_APPS=[]
+# GAS_PRICE=
+# RESTRICTED_LIST=
+# RESTRICTED_LIST_KEY=
+SHOW_MAINTENANCE_ALERT=false
+MAINTENANCE_ALERT_MESSAGE=
+SOURCIFY_INTEGRATION_ENABLED=false
+SOURCIFY_SERVER_URL=
+SOURCIFY_REPO_URL=
+CHAIN_ID=
+MAX_SIZE_UNLESS_HIDE_ARRAY=50
+HIDE_BLOCK_MINER=false
+DISPLAY_TOKEN_ICONS=false
+SHOW_TENDERLY_LINK=false
+TENDERLY_CHAIN_PATH=
+RE_CAPTCHA_SECRET_KEY=
+RE_CAPTCHA_CLIENT_KEY=
+RE_CAPTCHA_V3_SECRET_KEY=
+RE_CAPTCHA_V3_CLIENT_KEY=
+RE_CAPTCHA_DISABLED=false
+JSON_RPC=
+#API_RATE_LIMIT_DISABLED=true
+API_RATE_LIMIT_TIME_INTERVAL=1s
+API_RATE_LIMIT_BY_IP_TIME_INTERVAL=5m
+API_RATE_LIMIT=50
+API_RATE_LIMIT_BY_KEY=50
+API_RATE_LIMIT_BY_WHITELISTED_IP=50
+API_RATE_LIMIT_WHITELISTED_IPS=
+API_RATE_LIMIT_STATIC_API_KEY=
+API_RATE_LIMIT_UI_V2_WITH_TOKEN=5
+API_RATE_LIMIT_BY_IP=3000
+# API_RATE_LIMIT_HAMMER_REDIS_URL=redis://redis_db:6379/1
+# API_RATE_LIMIT_IS_BLOCKSCOUT_BEHIND_PROXY=false
+API_RATE_LIMIT_UI_V2_TOKEN_TTL_IN_SECONDS=18000
+FETCH_REWARDS_WAY=trace_block
+MICROSERVICE_SC_VERIFIER_ENABLED=true
+#MICROSERVICE_SC_VERIFIER_URL=http://smart-contract-verifier:8050/
+#MICROSERVICE_SC_VERIFIER_TYPE=sc_verifier
+MICROSERVICE_SC_VERIFIER_URL=https://eth-bytecode-db.services.blockscout.com/
```

Comment:
**question:**

for my understanding, what's behind this (in SUAVE's context)?

Reply:
This is an endpoint of [smart-contract verification service](https://github.com/blockscout/blockscout-rs/blob/main/smart-contract-verifier/README.md), which allows SUAVE users to verify smart-contracts through Blockscout's UI or API.